### PR TITLE
Upgrade to Decidim v0.20

### PR DESCRIPTION
**Pull request by LauraJaimeC**
CHANGELOG: https://github.com/decidim/decidim/blob/release/0.20-stable/CHANGELOG.md

Upgrade Decidim to v0.20

- [ ]  After deploy, must be checked email service is working correct.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -16,8 +16,8 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
 
+    # Default app time zone
     config.time_zone = 'Madrid'
```
Comment (tramuntanal) [inline config/application.rb:20]: we should remove this lines

File: config/environments/production.rb [hunk 2]
```diff
@@ -92,7 +92,8 @@
     password: Rails.application.secrets.smtp_password,
     domain: Rails.application.secrets.smtp_domain,
     enable_starttls_auto: Rails.application.secrets.smtp_starttls_auto,
-    openssl_verify_mode: 'none'
+    openssl_verify_mode: 'none',
+    ssl: true
   }
```
Comment (tramuntanal) [inline config/environments/production.rb:97]: Let's add a check in this PR's description to validate the email service after deploying this version.

File: config/initializers/decidim.rb [hunk 3]
```diff
@@ -13,8 +13,7 @@
   config.track_newsletter_links = true
   config.geocoder = {
     static_map_url: 'https://image.maps.cit.api.here.com/mia/1.6/mapview',
```
Comment (tramuntanal) [inline config/initializers/decidim.rb:15]: I think that the URL is also different

**Review by tramuntanal — COMMENTED** [review]
Good job @LauraJaimeC !

I've left just some details to review.

File: config/application.rb (hunk 1 above)
Comment (LauraJaimeC) [reply config/application.rb:20]: I think we should remove in 0.21 upgrade

**Review by tramuntanal — APPROVED** [review]

**Pull request merged**